workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -447,10 +447,8 @@ def _apply_postphysics_to_dycore_state(self) -> Diagnostics:
                     diagnostics[self._postphysics_stepper.net_moistening],
                     self._timestep,
                 )
-                diagnostics[TOTAL_PRECIP] = updated_state_from_tendency[TOTAL_PRECIP]
```

Comment:
grepping `total_precipitation` in fv3net shows that this diagnostic is weakly (non-breaking probably, but potentially confusing) assumed to be in the diagnostics by various other routines, e.g., it's in at least `workflows/prognostic_c48_run/runtime/diagnostics/default_diagnostics.py` and the end-to-end test diags specification. 

Seems reasonably in scope of this PR to make a clean break of this, presumably in favor of `total_precipitation_rate`. At least in the code -- the various test fixture data schema are inherently going to end up out of date so probably not worth it there.

Reply:
This diagnostics was deleted below, and so is never used. The tests would have revealed an error like this. In the first commit, I wrote out all the available diagnostics and check-summed them, before implementing this change in the second commit.

Reply:
Sure, that's fine. I guess this is another reminder why specifying default config in code is not a very maintainable approach.